pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -648,6 +648,16 @@ type ClusterAutoscaler struct {
 	// Verbosity allows CA to modify its log level (default: 2).
 	// +optional
 	Verbosity *int32 `json:"verbosity,omitempty" protobuf:"varint,14,opt,name=verbosity"`
+	// ExpanderConfig defines the configuration of the CA expander.
+	// +optional
+	ExpanderConfig *ExpanderConfig `json:"expanderConfig,omitempty" protobuf:"bytes,15,opt,name=expanderConfig"`
```

Comment:
Adapt `example/90-shoot.yaml`.

Generally, please revisit the comments of the previous PR reviews - some things are still missing.